Tests/Schema.NET.Test/Schema.NET.Test.csproj

```diff
@@ -23,6 +26,14 @@
     <PackageReference Include="StyleCop.Analyzers" PrivateAssets="All" Version="1.1.118" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(OS)' != 'Windows_NT' AND '$(TargetFramework)' == 'net472' ">

```

Comment:
To add `net472` support with mono, you should only need this https://github.com/RehanSaeed/Schema.NET/blob/c815daf10ed59e9328992e0f729b2ee23529e62b/Source/Schema.NET/Schema.NET.csproj#L45 NuGet package.

Reply:
I'm not actually sure what `Microsoft.TargetingPack.NETFramework.v4.7.2` is.

Reply:
Is it because of xUnit not supporting it? I think I remember something about this. This project used to target .NET 4.6.2 and I went through all kinds of gymnastics to get multi-tartgetting working.

Reply:
Yeah so I had just that initially however while it compiled, the tests failed with some internal xUnit error - was just trying to work around it. Its the first time I heard of `Microsoft.TargetingPack.NETFramework.v4.7.2` too but apparently its a thing that sounds like it does what we want though still not working.

Most of my testing experience is with MSTest/VSTest/whatever-they-call-it-these-days so I don't know about any quirks specific to xUnit.

Reply:
This is where I got the information about the targeting pack, from xUnit's own website: https://xunit.net/docs/getting-started/multi-target/non-windows

Reply:
Turns out the result is no better using [`TargetingPack`](https://ci.appveyor.com/project/RehanSaeed/schema-net/builds/29558592/job/mslwxfmsmpm37un0#L232) vs [`ReferenceAssemblies`](https://ci.appveyor.com/project/RehanSaeed/schema-net/builds/29566908/job/3ao46h9mld2rtdfp#L232).

Reply:
OK, got the tests running on .NET Framework. I've gone back to using `ReferenceAssemblies` because that is a whole lot neater and realised there was just one additional dependency needed. In 5c79c4e, I've added `Microsoft.TestPlatform.ObjectModel` and now the tests are running fine.

The failing test is known to fail for .NET Framework and should be fixed in #109.

Reply:
Please note that I've noticed chocolatey and AppVeyor have been having weird issues over the last few days, so those builds have been a bit flaky.

I think that targetting pack stuff is now superceded by the new `Microsoft.NETFramework.ReferenceAssemblies` package. All references to it seem a bit older.